ToolBox/BovespaConverter/config.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+    //----------------------------------------
+    // Configuration for the Bovespa Converter
+    //----------------------------------------
+
+    // Input folder in which zip raw files are saved
+    "input-folder": "C:\\Users\\Alexandre\\Documents\\Data",
+
+    // Output folder in which files will be saved (usually the Lean data folder)
+    "output-folder": "C:\\Users\\Alexandre\\Documents\\GitHub\\Lean\\Data",
```

Comment:
No, the input folder is the place the user saved the downloaded data. Since Bovespa delete the files to keep only two-years data, I would advise users to keep their copies and save them at folders "Equity", "Option" and "Future" inside "Data". I need to write a help file to explain that!

Ok, I will make 
"output-folder": "..\..\Data"

Thanks for the advise on factor files generation. I will look into it in future versions.